### PR TITLE
Fix: duplicates in query results

### PR DESCRIPTION
**Pull request by izquierdoalfredo**
Fix https://github.com/SeedCompany/cord-api-v3/issues/1156

### CODE REVIEW

File: src/core/database/query/lists.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ export function calculateTotalAndPaginateList(
     // .with(['node', 'total']) TODO needed?
     .call(sort, sortInput, order)
     .with([
-      `collect(node.id)[${(page - 1) * count}..${page * count}] as items`,
+      `collect(distinct node.id)[${(page - 1) * count}..${page * count}] as items`,
```
Comment (CarsonF) [inline src/core/database/query/lists.ts:17]: Haven't the nodes already been distinct-ed on line 12?

Comment (izquierdoalfredo) [reply src/core/database/query/lists.ts]: @CarsonF  Yes, but results still contain duplicates.
So I added last distinct operator

Comment (izquierdoalfredo) [reply src/core/database/query/lists.ts]: @CarsonF 
Do you agree with my solution?

Comment (CarsonF) [reply src/core/database/query/lists.ts]: Ok it looks like since the `sort` callback can do more matches this is needed and does make a difference. 👍

**Review by CarsonF — APPROVED** [review]
Thanks @izquierdoalfredo!

**Pull request merged**